Readme.md

```diff
@@ -52,15 +52,13 @@ ready for production use.
 
 ### Moya version vs Swift version.
 
-Because of the many Swift versions Moya supports, it might be confusing to
-find the version of Moya that you need. Below is a table that shows which version of Moya
-you should use for your Swift version.
+Below is a table that shows which version of Moya you should use for
+your Swift version.
 
 | Swift version | Moya version  |
 | ------------- | ------------- |
-| 3.X           | >= 8.0.0      |
-| 2.3           | 7.0.2 - 7.0.4 |
```

Comment:
I wouldn't remove old Swift versions, especially 2.3 which is still used in some projects.

Reply:
Thank you, I was biting my tongue on this 😓